src/Mesh/babylon.groundMesh.ts

```diff
@@ -5,6 +5,8 @@
         private _worldInverse = new Matrix();
         private _heightQuads: { slope: Vector2; facet1: Vector4; facet2: Vector4 }[];
         public _subdivisions: number;

```

Comment:
It does seem to be redundant.  I didn't dare to remove it, since I thought it could break something (public var).